.github/workflows/release.yml

```diff
@@ -98,22 +99,16 @@ jobs:
         gh release upload $RELEASE_TAG flank.sha256
 
     - name: Gradle Upload to MavenCentral
-      uses: gradle/gradle-build-action@v3.5.0
       env:
         PGP_SIGNING_KEY: ${{ secrets.PGP_SIGNING_KEY }}
-      with:
-        arguments: :test_runner:publishMavenJavaPublicationToMavenCentralRepository -PMVN_CENTRAL_USER=${{ secrets.MVN_CENTRAL_USER }} -PMVN_CENTRAL_PASSWORD=${{ secrets.MVN_CENTRAL_PASSWORD }}
+      run: ./gradlew :test_runner:publishMavenJavaPublicationToMavenCentralRepository -PMVN_CENTRAL_USER=${{ secrets.MVN_CENTRAL_USER }} -PMVN_CENTRAL_PASSWORD=${{ secrets.MVN_CENTRAL_PASSWORD }}
```

Comment:
You shouldn't use `${{ ...}}` in `run:`. 

Instead, best practice is: 

```yaml
env:
  MVN_CENTRAL_USER: ${{ secrets.MVN_CENTRAL_USER }}
  ... 
run: ... -PMVN_CENTRAL_USER="$MVN_CENTRAL_USER"... 
```

Where each secret/variable is filled into `env` and then used via an env variable.